src/test/debuginfo/repr-u128.rs

```diff
@@ -0,0 +1,30 @@
+// ignore-windows
+// ignore-tidy-linelength
+// min-system-llvm-version 8.0
+
+// compile-flags: -g -C no-prepopulate-passes
+
+// === LLDB TESTS ==================================================================================
```

Comment:
Why is GDB missing?

Reply:
@eddyb Sorry for being late. I fixed test.

Reply:
I'm not seeing `Some` or `None` in the tested output.

Reply:
Hmm, we cannot catch these now. This is a part of `repr-u128.out`.

```
print vals
((core::option::Option<repr_u128::Foo>, core::option::Option<repr_u128::Foo>)) $0 = (Option<repr_u128::Foo> { }, Option<repr_u128::Foo> { }) 
quit
None
```

Reply:
I really don't understand what's happening :/